account_manager/src/validator/deposit.rs

```diff
@@ -18,6 +19,7 @@ use web3::{
 pub const CMD: &str = "deposit";
 pub const VALIDATOR_FLAG: &str = "validator";
 pub const ETH1_IPC_FLAG: &str = "eth1-ipc";
+pub const ETH1_RPC_FLAG: &str = "eth1-rpc";
```

Comment:
Perhaps we should call this `eth1-http`, since it's technically RPC via IPC or HTTP?

Reply:
done!